### PR TITLE
[ams-18] Init of Amsterdam event 2018

### PR DESCRIPTION
**Pull request by thomaskrag**
Since we've done it for 5 years, why not 6?

### CODE REVIEW

**Comment by thomaskrag** [comment]
Just a side-note: This is most likely pending the creation of the 2018 email address.

**Comment by bridgetkromhout** [comment]
We need to update email lists and Slack at the same time as merging this; discussion pending in email.

File: data/events/2018-amsterdam.yml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    # linkedin: "https://www.linkedin.com/in/sallyfields"
+    # website: "https://mattstratton.com"
+    # image: "sally-fields.jpg"
+organizer_email: "organizers-amsterdam-2018@devopsdays.org" # Put your organizer email address here
+proposal_email: "proposals-amsterdam-2018@devopsdays.org" # Put your proposal email address here
+
+# List all of your sponsors here along with what level of sponsorship they have.
+# Check data/sponsors/ to use sponsors already added by others.
+sponsors:
+  - id: schubergphilis
```
Comment (bridgetkromhout) [inline data/events/2018-amsterdam.yml]: Do you already have this sponsor committed, or is this left over from a previous year? Thanks.

**Comment by bridgetkromhout** [comment]
Awaiting info about whether Daniel Paulus is participating for 2018 since he's not listed in this PR. Thanks.

File: data/events/2018-amsterdam.yml (hunk 1 above)
Comment (thomaskrag) [reply data/events/2018-amsterdam.yml]: This is a sponsor we have committed already. :)

**Comment by thomaskrag** [comment]
Fixed @bridgetkromhout :)

**Comment by bridgetkromhout** [comment]
Note, this can be merged if/when someone has time to set up email & slack for 2018 (I'll have time tomorrow).

**Comment by bridgetkromhout** [comment]
Okay, I made sure Slack and email were set up.

**Pull request merged**